### DISCUSSION TITLE
"bundleWorkAsShulker"

### DISCUSSION THREAD

**Discussion by ghost** [Minecraft-AMS/Carpet-AMS-Addition#76] [Ideas]
Removes the special item behaviour and let it work like a shulker box.

**Comment by wendavid552** [comment]
A bundle could contain multiple shulkers for more space. Why would you like to convert it back to a shulker?

**Reply by ghost** [reply]
Good that you mention it, maybe don't allow to put Shulkers into Bundles, easy solution. 

I know that it's possible to put Shulkers into Shulkers with another Carpet rule. But that's somehow too much. 

But putting Bundles into Shulker boxes seems better and helps to keep the overview.

**Reply by wendavid552** [reply]
So you mean bundle in shulkers, but no shulkers in a bundle?  or just make a bundle as a shulker with less slots? Isn't it the same now if you do not put shulkers inside?

**Reply by ghost** [reply]
- Bundle acts as a Shulker Box, but it doesn't allows for placing Shulkers inside

- I want to get rid of the special Item behaviour, see it more as an "early" Shulker Box availability and you can fill Bundles into Shulkers, to get even more storage optimization 😄

**Comment by 1024-byteeeee** [comment]
The bundle has not been officially added to the game yet, so we are not considering adding new features related to the bundle at the moment.

**Comment by ghost** [comment]
That's a fair point - If this item "ever" gets official released (😂), I will come back to it.
